lib/node_modules/@stdlib/stats/base/dists/planck/pmf/benchmark/benchmark.js

```diff
@@ -0,0 +1,81 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var discreteUniform = require( '@stdlib/random/array/discrete-uniform' );
+var randu = require( '@stdlib/random/base/randu' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var pkg = require( './../package.json' ).name;
+var pmf = require( './../lib' );
+
+
+// MAIN //
+
+bench( pkg, function benchmark( b ) {
+	var lambda;
+	var x;
+	var y;
+	var i;
+
+	x = discreteUniform( b.iterations, 0, 40 );
+
+	b.tic();
+	for ( i = 0; i < b.iterations; i++ ) {
+		lambda = ( randu()*10.0 );
```

Comment:
You need to move this out of the loop. Then use modulo arithmetic below (e.g., `i % x.length`, etc). As I mentioned elsewhere, see other recent changes to base stats dists packages where we've already been doing this work.